docs/eventing/sugar/README.md

```diff
@@ -0,0 +1,357 @@
+longer be updated.
+
+### Simple AutoTrigger Example
+
+When the following Knative Service is created:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello-sugar
+  namespace: candyland
+  labels:
+    eventing.knative.dev/autotrigger: enabled
+spec:
+  template:
+    spec:
+      containers:
+        - image: <some image>
+```
+
+The Sugar Controller will notice AutoTrigger is enabled for that resource, and
+create a trigger subscribing the Knative Service "hello-sugar" to all events
+send to the "default" Broker, the cluster will have a Trigger like:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: hello-sugar-abc123
+  namespace: candyland
+  ownerReferences:
+    - apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: hello-sugar
+spec:
+  broker: default
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: hello-sugar
+```
+
+If filters are added to the Service,
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello-sugar
+  namespace: candyland
+  labels:
+    eventing.knative.dev/autotrigger: enabled
+  annotations:
+    trigger.eventing.knative.dev/filter.attributes: |
+      [{"type":"gumdrops", "sticky":"yes"}]
+```
+
+A Trigger with filters will be replace the original:
+
+```yaml
+kind: Trigger
+...
+spec:
+  ...
+  filter:
+    attributes:
+      type: gumdrops
+      sticky: "yes"
+```
+
+### Multiple Triggers Example
+
+More than one Trigger could be configured in the filter annotation, when the
+following Knative Service is created:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello-sugar
+  namespace: candyland
+  labels:
+    eventing.knative.dev/autotrigger: enabled
+  annotations:
+    trigger.eventing.knative.dev/filter.attributes: |
+      [{"type":"gumdrops"}, {"subject":"lollipops"}]
+spec:
+  template:
+    spec:
+      containers:
+        - image: <some image>
+```
+
+Two Triggers will be created like:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: hello-sugar-abc123
+  namespace: candyland
+  ownerReferences:
+    - apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: hello-sugar
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: gumdrops
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: hello-sugar
+
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: hello-sugar-efg456
+  namespace: candyland
+  ownerReferences:
+    - apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: hello-sugar
+spec:
+  broker: default
+  filter:
+    attributes:
+      subject: lollipops
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: hello-sugar
+```
+
+### Broker Named Trigger Example
+
+Using the `autotrigger.eventing.knative.dev/broker` annotation, the Addressable
```

Comment:
```suggestion
Using the `autotrigger.eventing.knative.dev/broker` annotation, the Addressable resource
```
Not sure on the addition of resource, if "Addressable" is a resource type then we can ignore adding "resource"